boards/samd21-xpro/include/board.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2017 Travis Griggs <travisgriggs@gmail.com>
+ * Copyright (C) 2017 Dan Evans <photonthunder@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_samd21-xpro Atmel SAM D21 Xplained Pro
+ * @ingroup     boards
+ * @brief       Support for the Atmel SAM D21 Xplained Pro board.
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Atmel SAM D21 Xplained Pro
+ *              board
+ *
+ * @author      Travis Griggs <travisgriggs@gmail.com>
+ * @author      Dan Evans <photonthunder@gmail.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   xtimer configuration
```

Comment:
should be `@name` here and below (use `@name` with `@{ .. /** @} */` blocks, otherwise `@brief` is fine.